_posts/2019-08-17-intro-to-reports.adoc

```diff
@@ -0,0 +1,418 @@
+---
+compatible_cuba_versions: 7.0+
+compatible_java_versions: 8+
+project_id: cuba-petclinic-intro-to-reports
+permalink: introduction-to-reports
+---
+= Introduction to Reports and Document generation
+:showtitle:
+:sectlinks:
+:sectanchors:
+:page-navtitle: Introduction to Reports and Document generation
+:page-excerpt: In this guide the reports application component will be introduced and different ways to interact with it through the UI and programmatically will be shown
+:page-root: ../../../
+:project_id: cuba-petclinic-intro-to-reports
+:java_version: 1.8
+:cuba_version: 7.0
+
+
+CUBA applications can be enhanced to have a comprehensive report / document generation capability directly embedded into the application. The CUBA application component `reports` supports various file formats like `docx`, `xlsx`, `PDF`, `HTML` for templating as well as output file formats. Furthermore it contains management screens and provides APIs to interact with the report / document generation process programmatically.
```

Comment:
Let's call them add-ons instead of application components:
... The CUBA `reports` add-on supports ...

Also, use case consistently:
... formats like `DOCX`, `XLSX`, `PDF`, `HTML` ...

Reply:
👍